.travis.yml

```diff
@@ -7,15 +7,17 @@ addons:
       - wine
 
 env:
+  - DEBUG="electron-installer-windows" NODE_VERSION="12"
   - DEBUG="electron-installer-windows" NODE_VERSION="10"
```

Comment:
In the interests of CI resource consumption, I wonder if it's worth just testing against the oldest and newest supported LTS versions.

Reply:
I don't think it's a bad idea but I'm not sure how I feel about it. Especially for Appveyor, which ends up testing for each version, both x86, and x64.
Why not, I'll remove node 10 from testing, and use only 8 and 12. We should do the same in all other modules.

Reply:
> We should do the same in all other modules.

Yep, I was planning on it unless you had any objections.

Reply:
Not objections. I always had a feeling that we should test for each version we're using but when thinking carefully about it, there's really no good reason (that I can think of) of just using the oldest and newest LTS versions.